Lib/xml/etree/ElementTree.py

```diff
@@ -923,7 +923,7 @@ def _serialize_xml(write, elem, qnames, namespaces,
                             k,
                             _escape_attrib(v)
                             ))
-                for k, v in sorted(items):  # lexical order
+                for k, v in items:  # lexical order
```

Comment:
Done.

Reply:
I don't think this warrants a whatsnew entry.

Reply:
If other modules are to be changed as well (minidom, html modules, etc), it will need to be done in another PR.  I don't have the spare time for those right now.  Perhaps the other contributor on the tracker issue will continue to show an interest.

Reply:
At least remove sorting in other output methods in this module. And add tests for them.